src/test/scala/chisel3/tests/internal/VerilogCompilerSpec.scala

```diff
@@ -0,0 +1,28 @@
+// See LICENSE for license details.
+
+package chisel3.tests.internal
+
+import chisel3._
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage}
+import firrtl.VerilogCompiler
+import firrtl.options.TargetDirAnnotation
+import firrtl.stage.CompilerAnnotation
+import org.scalatest.{FreeSpec, Matchers}
+
+class VerilogCompilerSpec extends FreeSpec with Matchers {
```

Comment:
Does this really belong in testers2? This seems like a test that belongs in chisel base?

Reply:
Should be there, was an early testing artifact, will be removed on next commit